core/console/views/layout/layout.go

```diff
@@ -0,0 +1,20 @@
+package layout
```

Comment:
layouts now live in the layouts package, this includes things like base and site_frame